deploy.sh

```diff
@@ -50,6 +50,20 @@ user="admin"
 # docker: loop timeout is turned off by default
 timeout=-1
 
+function ctrlc_handler () {
+    # Check if cluster type is docker
+    if [[ "$cluster_type" == "docker" ]]; then
+        # Terminate the containers [kruize && grafana && prometheus && cadvisor]
+        docker_terminate
+    fi
+    # Exiting gracefully
+    exit 2
+}
+
+
+trap "ctrlc_handler" 2 # Handle SIGINT for cleaning up containers in docker case
```

Comment:
We can add the same handler for other signals as well (SIGHUP, SIGINT, SIGQUIT)

Reply:
Sure. Will be adding it.

Reply:
@dinogun made the changes as we discussed. Can i please have it reviewed ?